src/themes/default/modules.ts

```diff
@@ -4,6 +4,7 @@ import { Review } from '@vue-storefront/core/modules/review'
 import { Mailer } from '@vue-storefront/core/modules/mailer'
 import { Mailchimp } from '@vue-storefront/core/modules/mailchimp'
 import { Example } from '@vue-storefront/core/modules/module-template'
+import { extendedCart } from './modules/cart'
```

Comment:
I think for the sake of simplicity we can keep this in this extension in this file (like example) instead of making new dir (which may be confusing since this is actually a modification of module, not cart module) WDYT?